utility/src/main/java/org/oppia/util/parser/HtmlParsingAnnotaions.kt

```diff
@@ -0,0 +1,6 @@
+package org.oppia.util.parser
+
+import javax.inject.Qualifier
+
+/** Qualifier for injecting the application context. */
+@Qualifier annotation class ApplicationContext
```

Comment:
Let's not use this. I'd rather not duplicate the annotations.

You can just inject Context directly and it will be the application context. Once we resolve #59 we'll be able to fix this fully (there should already be TODO tracking this in the application component).

Reply:
removed.